setup.py

```diff
@@ -43,19 +43,22 @@
         # The following is for both SS and Envision
         "cloudpickle>=1.3.0,<1.4.0",
         # The following are for /envision
-        "tornado>=6.1",
+        "tornado>=5.1.1",
         "websocket-client>=1.2.1",
         # The following is used for imitation learning and envision
         "ijson>=3.1.4",
         # The following are for the /smarts/algorithms
-        "matplotlib>=3.4.3",
+        "matplotlib>=3.2.2",
         # The following are for /smarts/zoo and remote agents
-        "grpcio==1.32.0",
-        "protobuf>=3.19.1",
-        "PyYAML>=6.0",
+        "grpcio==1.32.0,<=1.44.0",
```

Comment:
Perhaps need to change the sign here as we have both a `==` sign and a `<=` sign here.

Reply:
I think we need to talk about it. `grpcio` seems to be the core compatibility issue alongside `gym`. Is there an alternative we can quickly leverage to make `grpcio` optional?